app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -252,12 +262,9 @@ DonorsChooseDonationController.prototype.postDonation = function(member, project
         try {
           logger.log('verbose', 'dc.requestToken POST user:%s body:%s', 
             donorPhone, body);
-          // Handles when we get 403/Forbidden but steps in when sufficient funds.
-          // if (typeof body !== 'object') {
-          //   self.endChatWithFail(member);
-          //   logger.error('dc.requestToken user:%s invalid body:%s', donorPhone, body);
-          //   return deferred.promise;
-          // }
+          if (typeof body !== 'object') {
+            logger.error('dc.requestToken user:%s invalid JSON:%s', donorPhone, body);
+          }
```

Comment:
The invalid JSON message still is returned for success too. Going to comment this out again for now :\

>2016-08-16T21:37:31.634020+00:00 app[web.1]: [31merror[39m: dc.requestToken user:[removed] invalid JSON:
2016-08-16T21:37:31.634031+00:00 app[web.1]: 
2016-08-16T21:37:31.634041+00:00 app[web.1]: 
2016-08-16T21:37:31.634041+00:00 app[web.1]: 
2016-08-16T21:37:31.634042+00:00 app[web.1]: 	
2016-08-16T21:37:31.634043+00:00 app[web.1]: 	
2016-08-16T21:37:31.634043+00:00 app[web.1]: 		
2016-08-16T21:37:31.634044+00:00 app[web.1]: 	
2016-08-16T21:37:31.634045+00:00 app[web.1]: 
2016-08-16T21:37:31.634045+00:00 app[web.1]: 
2016-08-16T21:37:31.634046+00:00 app[web.1]: 
2016-08-16T21:37:31.634046+00:00 app[web.1]: 
2016-08-16T21:37:31.634051+00:00 app[web.1]: 
2016-08-16T21:37:31.634052+00:00 app[web.1]: 
2016-08-16T21:37:31.634064+00:00 app[web.1]: 
2016-08-16T21:37:31.634065+00:00 app[web.1]: 
2016-08-16T21:37:31.634066+00:00 app[web.1]: 
2016-08-16T21:37:31.634083+00:00 app[web.1]: 
2016-08-16T21:37:31.634083+00:00 app[web.1]: 
2016-08-16T21:37:31.634083+00:00 app[web.1]: 
2016-08-16T21:37:31.634084+00:00 app[web.1]: 
2016-08-16T21:37:31.634085+00:00 app[web.1]: 
2016-08-16T21:37:31.634085+00:00 app[web.1]: 
2016-08-16T21:37:31.634086+00:00 app[web.1]: 
2016-08-16T21:37:31.634087+00:00 app[web.1]: 
2016-08-16T21:37:31.634087+00:00 app[web.1]: 
2016-08-16T21:37:31.634088+00:00 app[web.1]: 
2016-08-16T21:37:31.634090+00:00 app[web.1]: 
2016-08-16T21:37:31.634090+00:00 app[web.1]: 
2016-08-16T21:37:31.634092+00:00 app[web.1]: 
2016-08-16T21:37:31.634094+00:00 app[web.1]: {"statusCode":"0","statusDescription":"success","token":"[removed]"}
2016-08-16T21:37:31.634094+00:00 app[web.1]: 
2016-08-16T21:37:31.634094+00:00 app[web.1]: 
2016-08-16T21:37:35.490172+00:00 app[web.1]: [32minfo[39m: dc.postDonation POST success user:[removed] proposalId:2121637 donationId:4805064